packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx

```diff
@@ -219,9 +220,7 @@ export default {
 } as Meta<StoryControlProps>;
 
 const BottomSheetTemplate: ComponentStory<typeof BottomSheetComponent> = ({ ...args }) => {
-  // `!!isChramatic` is not readable hence disabling the eslint rule
-  // eslint-disable-next-line no-unneeded-ternary
-  const [isOpen, setIsOpen] = React.useState(isChromatic() ? true : false);
+  const [isOpen, setIsOpen] = React.useState(isReactNative() ? false : isChromatic());
```

Comment:
Interesting! Didn't know about isChromatic. Can use this in Dropdown as well 🥳

Reply:
Yeah, though it fails on RN, so make sure to add isReactNative check also.